.github/workflows/verify_newproject_script.yml

```diff
@@ -0,0 +1,37 @@
+name: Verify newproject script
+
+on:
+  pull_request:
+    types: [ opened, edited, reopened, synchronize ]
```

Comment:
I think we can omit the `edited` because the edited type will trigger to run this workflow when we update the title or description of the PR. IMO just opened, reopened, and synchronize are sufficient.

Reply:
@lydiasama Make sense 👍

Reply:
Oh, what if the pull request gets `updated`, will it be triggered again? I think it should 🙏

Reply:
@Tuubz 👍

Reply:
I'm surprised to see that there's no option for:

```
on:
  pull_request:
    types: [ updated ]
```

But then I had a look in to [this documentation](https://docs.github.com/en/enterprise-server@3.2/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-29):

> synchronize: Triggered when a pull request's head branch is updated. For example, when the head branch is updated from the base branch, when new commits are pushed to the head branch, or when the base branch is changed.

Do you think `synchronize` will be triggered whenever the pull request is updated? 🤔

Reply:
> Do you think synchronize will be triggered whenever the pull request is updated? 🤔

`updated` in your mind is new commits pushed, right? If so, I have the same idea about `synchronize`, the event will be triggered if there is new commits pushed on the PR.

Reply:
Updated could be: New commit, rebase then force push, append old commit then force push, ...

If `synchronize` does it for us, then the following code might be redundant:

```
push:
    paths:
      - 'newproject.sh'
    branches:
      - 'feature/**'
      - 'chore/**'
      - 'bug/**'
```

Please correct me if I misunderstood 🙏

Reply:
@Tuubz So basically, we don't need to trigger the workflow on `pull_request` if the `newproject.sh` is not in the list of change files 🤔  What do you think if we just need to trigger on `push`?

Reply:
Following this answer: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

@Tuubz So from our case, the event triggers should be like this:

```
on:
  pull_request:
    types: [ opened, reopened, synchronize ]
    branches: [ develop ]
  push:
    branches:
      - 'feature/**'
      - 'chore/**'
      - 'bug/**'
```

Reply:
Hmmm, from my understanding `synchronize` is the same as:

```
push:
    branches:
      - 'feature/**'
      - 'chore/**'
      - 'bug/**'
```

Because `synchronize` will trigger whenever there's an update the pull request. 🤔 

Reference: [StackOverflow](https://stackoverflow.com/questions/65093647/in-github-actions-how-can-i-trigger-on-push-but-only-if-a-pr-is-active)

Reply:
@Tuubz Hmm... So we only want to trigger only not both. I think I would get rid of

```
push:
    branches:
      - 'feature/**'
      - 'chore/**'
      - 'bug/**'
```

🚀